### ISSUE TITLE
Multi-App Kiosk Provisioning Error

### ISSUE THREAD

**Issue by slunat** [MicrosoftDocs/windows-itpro-docs#725]
I have followed the tutorial exactly and validated my XML against the XSD. It all verified correctly so my formatting must be correct. However, the provisioning package always fails to add properly. I am trying to add the package to the same machine I created the XML on and it still wont work. The error message I get is "Error during package installation". When clicking on details the only extra explanation I get is "A provisioning failure has occurred".
I have tried two different packages.
THe first package just linked to a new profile I created call kiosk, but after the package failed to add, I checked the kiosk profile and no changes were made to it at all, which was to be expected as the package had failed to add.
The second package I created, I added into the configuration in ICD to create a new account called TestUser2 and changed my XML to apply the profile to that account instead. When trying to add this provisioning package, it managed to create a new user on my system. However, it failed to add the package successfully and no changes were made on the account.
Could this be my XML that is the issue? It always verifies correctly against the XSD so it should be right.
I am using Windows 10 Home 1709
Please see my attached project files
[Provisioning Package Files.zip](https://github.com/MicrosoftDocs/windows-itpro-docs/files/1914521/Provisioning.Package.Files.zip)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 78038d67-04a1-f4ae-6a69-98e157688a31
* Version Independent ID: c86b4460-e8be-8537-a6c8-3cb094ef14a3
* Content: [Create a Windows 10 kiosk that runs multiple apps (Windows 10)](https://docs.microsoft.com/en-us/windows/configuration/lock-down-windows-10-to-specific-apps)
* Content Source: [windows/configuration/lock-down-windows-10-to-specific-apps.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/configuration/lock-down-windows-10-to-specific-apps.md)
* Service: **unspecified**
* GitHub Login: @jdeckerMS
* Microsoft Alias: **jdecker**

**Comment by jdeckerms** [comment]
Kiosk mode (assigned access) is only supported for Windows 10 Pro, Enterprise, and Education. I'll update the topic to make that clear. https://www.microsoft.com/en-us/windows/compare

**Comment by wm-marting** [comment]
I ahve also gone thru the tutorial and double/triple-verified all steps and am also seeing "A provisioning failure has occurred" after I add the package.  I tried to follow the steps in "Troubleshooting provisioning issues", which pointed me at a specific part of the event log, but there was no additional information about my problem there.  I have tried on both Windows 10 Pro and Windows 10 Enterprise, and have tried with both classic and UWP apps.
